python/cuml/svm/svm.pyx

```diff
@@ -138,7 +138,8 @@ class SVC(Base):
         tol : float (default = 1e-3)
             Tolerance for stopping criterion.
         cache_size : float (default = 200 MiB)
-            Size of the kernel cache n MiB
+            Size of the kernel cache during training in MiB. This is also used
```

Comment:
It would be a good idea to elaborate on this property a little further so that users might know when they would change it and the implications it has on the memory footprint / performance of the algorithm.

Reply:
Done.